[2025-02-08T19:01:11.678+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:01:11.695+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2025-01-17T00:00:00+00:00 [queued]>
[2025-02-08T19:01:11.705+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2025-01-17T00:00:00+00:00 [queued]>
[2025-02-08T19:01:11.706+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:01:11.803+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2025-01-17 00:00:00+00:00
[2025-02-08T19:01:11.812+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1033) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:01:11.814+0000] {standard_task_runner.py:72} INFO - Started process 1049 to run task
[2025-02-08T19:01:11.812+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2025-01-17T00:00:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpwx6h_9m8']
[2025-02-08T19:01:11.816+0000] {standard_task_runner.py:105} INFO - Job 133: Subtask generate_account_dim_data
[2025-02-08T19:01:11.873+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2025-01-17T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:01:11.964+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-17T00:00:00+00:00'
[2025-02-08T19:01:11.965+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:01:11.993+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T19:01:11.999+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:01:12.013+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:01:12.014+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2025-01-17T00:00:00+00:00, execution_date=20250117T000000, start_date=20250208T190111, end_date=20250208T190112
[2025-02-08T19:01:12.069+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:01:12.079+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:42:23.290+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:42:23.304+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2025-01-17T00:00:00+00:00 [queued]>
[2025-02-08T20:42:23.314+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2025-01-17T00:00:00+00:00 [queued]>
[2025-02-08T20:42:23.314+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:42:23.325+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2025-01-17 00:00:00+00:00
[2025-02-08T20:42:23.333+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2025-01-17T00:00:00+00:00', '--job-id', '363', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpjn2ja9f6']
[2025-02-08T20:42:23.336+0000] {standard_task_runner.py:105} INFO - Job 363: Subtask generate_account_dim_data
[2025-02-08T20:42:23.338+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=201) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:42:23.339+0000] {standard_task_runner.py:72} INFO - Started process 208 to run task
[2025-02-08T20:42:23.383+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2025-01-17T00:00:00+00:00 [running]> on host 86932746817d
[2025-02-08T20:42:23.462+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-17T00:00:00+00:00'
[2025-02-08T20:42:23.464+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:42:23.484+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T20:42:23.484+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:42:23.492+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:42:23.493+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2025-01-17T00:00:00+00:00, execution_date=20250117T000000, start_date=20250208T204223, end_date=20250208T204223
[2025-02-08T20:42:23.514+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:42:23.529+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:42:23.532+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
